src/components/scenes/courses/CourseManagement/components/GuideManagement/GuideManagement.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { Container, Typography } from '@material-ui/core';
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+import Grid from '@material-ui/core/Grid';
+import ExerciseManagement from '../ExerciseManagement/ExerciseManagement';
+import styles from '../ExerciseManagement/ExerciseManagement.module.sass';
+
+export default class GuideManagement extends Component {
+  componentDidMount() {
+    this.updateExercises();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { location } = this.props;
+    if (location !== prevProps.location) {
+      this.updateExercises();
+    }
+  }
+
+  updateExercises() {
+    const { getExercises, courseId, guide } = this.props;
+    console.log(guide);
+    if (guide) getExercises({ courseId, guideId: guide.guideId });
+  }
+
+  render() {
+    const {
+      courseId, guide, exercises, showAddExerciseModal
+    } = this.props;
+    if (!guide) {
+      // TODO
+      return 'Por favor selecciona una guia';
+    }
+    const { guideId } = guide;
+
+    return (
+      <Container className={styles.exerciseManagement}>
+
+        <Typography align="center" variant="h6" className={styles.guideTitle}>
+          {guide.name}
+        </Typography>
+
+        {/* TODO: filter
+                <div> */}
+        {/*    Filtro de ejercicios por si tiene muchos */}
+        {/* </div> */}
```

Comment:
I would remove this code because we will implement pagination in another sprint (mentira)

Reply:
Deleted!